src/components/ArticleView/CommentEntry.jsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+import { postCommentByArticleId } from '../../../api';
+
+const CommentEntry = ({ article_id, onCommentPosted }) => {
+    const [userComment, setUserComment] = useState('');
+
+    function handlePostComment() {
+        postCommentByArticleId(article_id, {
+            username: 'cooljmessy',
```

Comment:
absolutely fine to hard code this here for now - I would let the user know who they are posting a comment as though. 

if you get upto implementing useContext then you can share this user around your whole app and grab it from there (it could still be hardcoded, then later you could have some kind of netflix style log in where people can select different users if they like?)